superset-frontend/spec/javascripts/explore/components/QueryAndSaveBtns_spec.jsx

```diff
@@ -50,7 +50,7 @@ describe('QueryAndSaveButtons', () => {
 
     it('renders buttons with correct text', () => {
       expect(wrapper.find(Button).contains('Run')).toBe(true);
-      expect(wrapper.find(Button).contains(' Save')).toBe(true);
+      expect(wrapper.find(Button).contains('Save')).toBe(true);
```

Comment:
I'm unsure why GH is showing these lines as changed, as I haven't touched them (the branch is based on `apache:feature/filter-p0` and is up-to-date)